common/test/opentelemetry/common/utilities_test.rb

```diff
@@ -19,19 +19,19 @@ def shutdown(timeout: nil); end
 
   describe '#untraced?' do
     it 'returns true within an untraced block' do
-      assert_equal(true, common_utils.untraced { common_utils.untraced? })
+      assert(common_utils.untraced { common_utils.untraced? })
```

Comment:
These suggestions from minitest can sometimes be false positives. I usually keep `assert_equal` when I want to validate the return value is `true` instead of any truthy value. Given the description of the test, this assertion may be a good candidate to keep using `assert_equal`. 

If you agree, the test below this one should also be reverted.